domain/src/main/java/org/oppia/domain/topic/StoryProgressController.kt

```diff
@@ -17,6 +17,8 @@ const val TEST_EXPLORATION_ID_1 = "test_exp_id_1"
 const val TEST_EXPLORATION_ID_2 = "test_exp_id_2"
 const val TEST_EXPLORATION_ID_3 = "test_exp_id_3"
 const val TEST_EXPLORATION_ID_4 = "test_exp_id_4"
+const val FRACTIONS_EXPLORATION_ID_0 = "0"
```

Comment:
Are these IDs correct? Shouldn't we use the actual exploration IDs for this & the next one to ensure they link properly to topic data?

Reply:
These were the IDs that were set in the topic that's why I did it this way. I can otherwise change the exploration IDs set in the topic

Reply:
Ah. My main goal was to make sure they were the same. The actual value isn't important.